### DISCUSSION TITLE
请问可以在增加标题的层数吗？

### DISCUSSION THREAD

**Discussion by nicholasyellow** [windingwind/zotero-better-notes#322] [Ideas]
请问可以在增加标题的层数吗？
目前只能支持3级标题，能否开发支持四级、五级标题呢？

**Comment by Ziilongg** [comment]
我也非常需要这个功能

**Comment by windingwind** [comment]
支持1-6级

**Reply by nicholasyellow** [reply]
是最新版本吗，我下载的只有1-3级

**Reply by Ziilongg** [reply]
> 支持1-6级

我在pad上看到的按钮里是可以有六级的，说明zotero内置就支持六级。
但是PC上的按钮里只有三级，不知道怎么调出来

**Reply by Ziilongg** [reply]
> 是最新版本吗，我下载的只有1-3级

我找到设置更多标题层级的办法了，就是通过MD语法自己打出来，#+空格 
吃了没有文化的亏

**Reply by zhangzzx6482** [reply]
能分享下代码不

**Reply by windingwind** [reply]
ctrl+shift+1-6

**Comment by zhangzzx6482** [comment]
您发的邮件已经收到。
